src/main/java/interface_adapter/recipeCollection/RecipeCollectionViewModel.java

```diff
@@ -1,8 +1,10 @@
-package interface_adapter.recipe_collection;
```

Comment:
This is a great catch! Adhering to camel case will not annoy Checkstyle and follows Java synthax. Feedback I would recommend is to lay out a diagram with all your package and class names and etc. to make it easier and ease the burden of refactoring all over again.